.github/workflows/nodeCI.yml

```
@@ -0,0 +1,23 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [dev]
+  pull_request:
+    branches: [dev]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm run build --if-present
```

@sjlevens CI builds are failing because its treating warnings as errors. I missed setting `CI` to `false`

```suggestion
      - run: npm run build 
        env:
            CI: false
```